### ISSUE TITLE
coco_val in my dataset

### ISSUE THREAD

**Issue by gtfaiwxm** [zylo117/Yet-Another-EfficientDet-Pytorch#315]
training my dataset 50 epochs, loss is 0.2. When I use coco_val , the values are -1.
